cfgov/v1/models/organisms.py

```diff
@@ -1,4 +1,8 @@
 from wagtail.wagtailcore import blocks
+from wagtail.wagtailcore.fields import RichTextField, StreamField
+from wagtail.wagtailadmin.edit_handlers import FieldPanel, StreamFieldPanel
+
+from . import molecules
 
 
 class Well(blocks.StructBlock):
```

Comment:
The well right now should just be one rich text field. The macro originally had a template path for demo-ing purposes, but content should just see that rich text field.

![image](https://cloud.githubusercontent.com/assets/1860176/11284312/0bc7b85e-8ed7-11e5-83e0-9db66be9409e.png)